### PR TITLE
added support for Slovenian language

### PR DESCRIPTION
**Pull request by kjesenski**
added support for Slovenian language

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.25.0 / 2020-12-07
```
Comment (EvanHahn) [inline HISTORY.md:1]: Please change this to:

```suggestion
# Unreleased
```

File: package-lock.json [hunk 2]
```diff
@@ -419,6 +419,7 @@
             "has": "^1.0.3",
```
Comment (EvanHahn) [inline package-lock.json:419]: Please revert these changes. (`git checkout main package-lock.json`)

File: humanize-duration.js [hunk 3]
```diff
@@ -900,6 +900,91 @@
...
+        } else {
+          return "sekund";
+        }
+      },
+      ms: function (c) {
+        if (c % 10 === 1) {
+          return "milisekunda";
+        } else if (c % 100 === 2) {
+          return "milisekundi";
+        } else if (c % 100 === 3 || c % 100 === 4 || Math.floor(c) !== c) {
```
Comment (EvanHahn) [inline humanize-duration.js:980]: Do the other units need this `Math.floor` check?

**Review by EvanHahn — CHANGES_REQUESTED** [review]
Thank you! This pull request looks good but needs some small changes.

File: HISTORY.md (hunk 1 above)
Comment (kjesenski) [reply HISTORY.md:1]: I'll change this

File: package-lock.json (hunk 2 above)
Comment (kjesenski) [reply package-lock.json:419]: I'll revert this

File: humanize-duration.js (hunk 3 above)
Comment (kjesenski) [reply humanize-duration.js:980]: How I understand Math.floor is needed only if we use decimal numbers and how I see we use decimal numbers only with milliseconds. Or larger units can have decimal values?

Comment (EvanHahn) [reply humanize-duration.js:980]: Larger units can have decimal values by changing the `units` option.

Comment (kjesenski) [reply humanize-duration.js:980]: Ok, thanks then I'll change that and run some tests, and when everything is working I'll amend.

**Comment by kjesenski** [comment]
@EvanHahn I added the Math.floor() where was needed, now everything is covered. And I also changed the other issue what I had

**Review by EvanHahn — APPROVED** [review]

**Comment by EvanHahn** [comment]
Thanks! This has been published in `humanize-duration@3.25.0`.
